src/main.ml

```diff
@@ -1,23 +1,17 @@
 open! Lplib
 
-(*
-(** [sig_of_file f] returns the signature of the file path [f]. *)
-let sig_of_file : string -> Sign.t = fun fname ->
-  let mp = Files.module_path fname in
-  begin try Compile.compile true mp
-    with
-        Console.Fatal(None, msg) -> Console.exit_with "%s" msg
-      | Console.Fatal(Some(p), msg) -> Console.exit_with "[%a] %s" Pos.print p msg end;
-  Files.PathMap.find mp Sign.(Timed.(!loaded))
-*)
 let spec =
-  Arg.align []
+  Arg.align 
+  [
+    ("--signature", Arg.Set_string Deskolem.signature_name, "Set the name of the signtaure file");
+    ("--package", Arg.Set_string Deskolem.package_name, "Set the name of the lambdapi package")
+  ]
 
 let _ =
-  let usage = Printf.sprintf "Usage: %s [OPTIONS] [FILES]" Sys.argv.(0) in
+  let usage = Printf.sprintf "Usage: %s --signature [SIGNATURE_NAME] [FILES]" Sys.argv.(0) in
   let files = ref [] in
   Arg.parse spec (fun s -> files := s :: !files) usage;
   Common.Library.set_lib_root None;
   files := List.rev !files;
-  let process_file s = s |> Handle.Compile.compile_file |> Deskolem.test in
+  let process_file s = Deskolem.main s (s |> Handle.Compile.compile_file) in
```

Comment:
Why do you call compile_file? Skonverto should generate LP files only, not call LP to compile them.